### ISSUE TITLE
Awesome

### ISSUE THREAD

**Issue by younes0** [chmllr/WikiZen#1]
Just spent hours looking for a wiki supporting dropbox/markdown and came across your repo.
I would love to use it but there's no indications if it's product ready

**Comment by chmllr** [comment]
Hi, did you visit https://wikizen.click ? Or what do you mean with product ready?

**Comment by younes0** [comment]
Hey! sorry I meant production ready 
Well I don't see any setup instructions, I'd like to use it locally so I can use the dropbox persistence

**Comment by chmllr** [comment]
I see. Well it is not my first priority to provide WikiZen as an installable package. And anyway, why would you want to run it locally? There is no backend, it is just a static JS site and your data is stored in _your_ Dropbox. So what would you win running it locally? You'll definitely loose all updates and bug fixes.  

In genral you can take a look into the source code and into Makefile to figure out how to run it. You'll also need to run a static http server in the folder with index.html and other files.

I'd also appreciate any contribution to installation documentation.

**Comment by younes0** [comment]
You're right. I think I misunderstood something since the dropbox persistence didn't work for me, I thought that you made it intentionally on your website. Did I miss something there?

**Comment by chmllr** [comment]
Could you please describe what problems exactly do you experience with the Dropbox persistence?

**Comment by younes0** [comment]
Youu're using the Dropbox Datastore API, hence the missing markdown files in my Dropbox folder, got it finally!

Why not making another Dropbox persistence that uses the Core API to write Markdown files in the Dropbox folder?

**Comment by chmllr** [comment]
Well in this case WikiZen would be potentially able to modify _any_ files in your Dropbox. I'm not sure if I'd use such an app by myself.

**Comment by chmllr** [comment]
But what notice is that maybe the description of WikiZen is a little bit misleading. I think I'll improve the description later.

**Comment by younes0** [comment]
You can restrict to your app specific folder: https://www.dropbox.com/developers/reference/devguide#app-permissions 
> App folder

>A dedicated folder named after your app is created within the Apps folder of a user's Dropbox. Your app gets read and write access to this folder only and users can provide content to your app by moving files into this folder.Your app can also read and write datastores using the Datastore API.

**Comment by chmllr** [comment]
Thanks a lot. I missed this somehow. I will test this, because I'd definitely prefer this over datastores.

**Comment by younes0** [comment]
You're very welcome
